cli/tools/test/fmt.rs

```diff
@@ -74,3 +80,136 @@ pub fn format_test_error(js_error: &JsError) -> String {
     .to_string();
   format_js_error(&js_error)
 }
+
+pub fn format_sanitizer_diff(diff: RuntimeActivityDiff) -> Vec<String> {
+  let mut output = format_sanitizer_accum(diff.appeared, true);
+  output.extend(format_sanitizer_accum(diff.disappeared, false));
+  output.sort();
+  output
+}
+
+fn format_sanitizer_accum(
+  activities: Vec<RuntimeActivity>,
+  appeared: bool,
+) -> Vec<String> {
+  let mut accum = HashMap::new();
+  for activity in activities {
+    let item = format_sanitizer_accum_item(activity);
+    accum.entry(item).or_insert(0).add_assign(1);
+  }
+
+  let mut output = vec![];
+  for ((item_type, item_name), count) in accum.into_iter() {
+    if item_type == RuntimeActivityType::Resource {
+      // TODO(mmastrac): until we implement the new timers and op sanitization, these must be ignored in this path
+      if item_name == "timer" {
+        continue;
+      }
```

Comment:
What's the consequence of it? Are timers ignored from sanitization altogether?

Reply:
Nope, they are picked up via op sanitization. Timers technically leak both an op and a resource and we were only showing the op before as a side effect of how things were implemented